<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QvisMoviePreferencesDialog</class>
 <widget class="QDialog" name="QvisMoviePreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>467</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>10</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Movie Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="FileGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="FileLayout">
      <item>
       <layout class="QHBoxLayout" name="FileNameLayout">
        <item>
         <widget class="QLabel" name="FileNameLabel">
          <property name="text">
           <string>File name</string>
          </property>
          <property name="buddy">
           <cstring>FileNameText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="FileNameText">
          <property name="text">
           <string>fluorender</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="EmbedInProjectFoldercheckBox">
        <property name="text">
         <string>Embed all files in the project folder</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="DirectoryLayout">
        <item>
         <widget class="QLabel" name="DirectoryLabel">
          <property name="text">
           <string>Directory</string>
          </property>
          <property name="buddy">
           <cstring>DirectoryText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DirectoryText">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="DirectoryButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="FileParameterGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="FileParameterLayout">
      <item>
       <layout class="QHBoxLayout" name="FileTypeLayout">
        <item>
         <widget class="QLabel" name="FileTypeLabel">
          <property name="text">
           <string>File type:</string>
          </property>
          <property name="buddy">
           <cstring>FileTypeComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="FileTypeComboBox">
          <item>
           <property name="text">
            <string>Tiff</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mov</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="FileTypeSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="LZWCompressionCheckBox">
        <property name="text">
         <string>LZW (Lempel-Ziv-Welch) Compression</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SaveAlphaChannelCheckBox">
        <property name="text">
         <string>Save alpha channel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SaveFloatChannelCheckBox">
        <property name="text">
         <string>Save float channel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="BitRateLayout">
        <item>
         <widget class="QLabel" name="BitRateLabel">
          <property name="text">
           <string>Bit rate</string>
          </property>
          <property name="buddy">
           <cstring>BitRateSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="BitRateSpinBox">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>50.000000000000000</double>
          </property>
          <property name="value">
           <double>20.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="BitRateUnitsLabel">
          <property name="text">
           <string>Mbps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="BitRateSpacer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>10</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="EstimatedSizeLayout">
        <item>
         <widget class="QLabel" name="EstimatedSizeLabel">
          <property name="text">
           <string>Estimated size:</string>
          </property>
          <property name="buddy">
           <cstring>EstimatedSizeText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="EstimatedSizeText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="EstimatedSizeUnitsLabel">
          <property name="text">
           <string>MB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="EstimatedSizeSpacer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ScaleOutputImageGroupBox">
     <property name="title">
      <string>Scale output image</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="ScaleOutputImageLayout">
      <item>
       <widget class="QSlider" name="ScaleOutputImageSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ScaleOutputImageSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>FileNameText</tabstop>
  <tabstop>EmbedInProjectFoldercheckBox</tabstop>
  <tabstop>DirectoryText</tabstop>
  <tabstop>DirectoryButton</tabstop>
  <tabstop>FileTypeComboBox</tabstop>
  <tabstop>BitRateSpinBox</tabstop>
  <tabstop>EstimatedSizeText</tabstop>
  <tabstop>ScaleOutputImageSlider</tabstop>
  <tabstop>ScaleOutputImageSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>LZWCompressionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>LZWCompressionToggled  (bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveAlphaChannelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>SaveAlphaChannelToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveFloatChannelCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>SaveFloatChannelToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EmbedInProjectFoldercheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>EmbedInProjectFolderToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DirectoryButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>DirectoryButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleOutputImageSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>ScaleOutputImageSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleOutputImageSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>ScaleOutputImageSpinBoxChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FileNameText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>FileNameTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FileTypeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>FileTypeIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ScaleOutputImageGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>ScaleOutputImageToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BitRateSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>BitRateSpinBoxValueChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>CancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OkButton</sender>
   <signal>clicked()</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>OkButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DirectoryText</sender>
   <signal>returnPressed()</signal>
   <receiver>QvisMoviePreferencesDialog</receiver>
   <slot>DirectoryTextReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>LZWCompressionToggled  (bool)</slot>
  <slot>SaveAlphaChannelToggled(bool)</slot>
  <slot>SaveFloatChannelToggled(bool)</slot>
  <slot>ScaleOutputImageToggled(bool)</slot>
  <slot>ScaleOutputImageSliderChanged(int)</slot>
  <slot>ScaleOutputImageSpinBoxChanged(double)</slot>
  <slot>EmbedInProjectFolderToggled(bool)</slot>
  <slot>FileNameTextChanged(QString)</slot>
  <slot>FileNameFamilyToggled(bool)</slot>
  <slot>DirectoryTextChanged(QString)</slot>
  <slot>DirectoryButtonClicked()</slot>
  <slot>FileTypeIndexChanged(int)</slot>
  <slot>BitRateSpinBoxValueChanged(double)</slot>
  <slot>CancelButtonClicked()</slot>
  <slot>OkButtonClicked()</slot>
  <slot>DirectoryTextReturnPressed()</slot>
 </slots>
</ui>
